test/zippy/mzcompose.py

```diff
@@ -194,3 +202,47 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
         )
         print("Running test...")
         test.run(c)
+
+
+def toxiproxy_start(c: Composition) -> None:
+    c.up("toxiproxy")
+
+    port = c.default_port("toxiproxy")
+    r = requests.post(
+        f"http://localhost:{port}/proxies",
+        json={
+            "name": "cockroach",
+            "listen": "0.0.0.0:26257",
+            "upstream": "cockroach:26257",
+            "enabled": True,
+        },
+    )
+    assert r.status_code == 201, r
+    r = requests.post(
+        f"http://localhost:{port}/proxies",
+        json={
+            "name": "minio",
+            "listen": "0.0.0.0:9000",
+            "upstream": "minio:9000",
+            "enabled": True,
+        },
+    )
+    assert r.status_code == 201, r
+    r = requests.post(
+        f"http://localhost:{port}/proxies/cockroach/toxics",
+        json={
+            "name": "cockroach",
+            "type": "latency",
+            "attributes": {"latency": 0, "jitter": 100},
```

Comment:
I think this needs to be made configurable so that only selected Zippy workloads run with this.

Reply:
I was thinking so first too, but which ones? Duplicating all of them would be too much I think.

Reply:
You need only the KafkaUserTables at this time , I think as the persist-txn code currently kicks in for tables only.